odtp/cli/execution.py

```diff
@@ -107,5 +113,45 @@ def streamlogs(
         sys.exit()
 
 
+@app.command()
+def delete(
+    execution_name: str = typer.Option(
+        None, "--execution-name", help="Specify the name of the execution"
+    ),
+    execution_id: str = typer.Option(
+        None, "--execution-id", help="Specify the ID of the execution"
+    ),
+    project_path: str = typer.Option(
+        None, "--project-path", help="Specify the path for the execution"
+    ),
+    keep_project_path: bool = typer.Option(
+        True, "--keep-project-path", help="Keep the project directory after deleting contents"
+    ),
+):
+    try:
+        if execution_id is None and execution_name is None:
+            raise typer.Exit("Please provide either --execution-name or --execution-id")
+
+        if execution_name:
+            execution_id = db.get_document_id_by_field_value("title", execution_name, db.collection_executions)
+
+        # S3
+        s3_keys = db.get_all_outputs_s3_keys(execution_id)
+        s3M = s3Manager()
+        s3M.deletePaths(s3_keys)
+
+        # DB
+        db.delete_execution(execution_id)
+        
+        # Folders
+        if project_path:
+            odtp_env.delete_folder(project_path, keep_project_path=keep_project_path) 
+
+    except Exception as e:
+        log.error(f"ERROR: Delete execution failed: {e}")       
+        raise typer.Abort()  
+    else:
+        log.info("SUCCESS: execution has been deleted")
```

Comment:
Add print statements in the cli command, since the log statement will only go to file

Reply:
Thanks for the rewiew @sabinem, I see the logic now. 

I'll follow this, but this should be change as In the future I would like to use print statements in the CLI to pipe CLI commands (Not implemented yet). 

Ideally only log.X() should go to the log file, otherwise it's a bit confusing.